### ISSUE TITLE
change variables

### ISSUE THREAD

**Issue by kailunn** [VeritasYin/STGCN_IJCAI-18#29]
Hi, thank you for sharing this neural network. I'm trying to apply this to the ChemE data and actually our data are kinda different! I though that I need to rewrite the data pre-processing part in main.py and like  weight matrix definition in math_graph. I will change other basic arguments as well (mentioned in main.py). 
I wonder that should I rewrite other part of this code which I've missed or ignored for the new data?
bc I did rewrite it actually, ummm.. it still didn't work! 
Thanks for patience :)

**Comment by ahariri13** [comment]
@kailunn I kind of have the same problem :) I have a different kind of dataset, but the same approach works. For now I believe the number of channels should be changed depending on how many variables you're trying to predict (in this case 1 only, i.e the speed). For sure you should also change the loss function in base_model. 

For me, it is still unclear how the model_inference process is working, in addition to the difference between "merge" mode and "sep" mode in trainer.py. For instance i am not sure what line 72 in trainer.py does (As i mentioned in one issue 2 days ago). If you get to know the answers for these please let me know :) 

Cheers

**Comment by VeritasYin** [comment]
#28
